testonly/hammer/hammer.go

```diff
@@ -412,8 +412,12 @@ func (s *hammerState) chooseInvalid(ep MapEntrypointName, prng *rand.Rand) bool
 }
 
 func (s *hammerState) chooseLeafCount(prng *rand.Rand) int {
-	delta := 1 + s.cfg.MaxLeaves - s.cfg.MinLeaves
-	return s.cfg.MinLeaves + prng.Intn(delta)
+	return pickIntInRange(s.cfg.MinLeaves, s.cfg.MaxLeaves, prng)
```

Comment:
Since this method now seems to be used only once in `hammerState`, I suggest inlining it.

Reply:
Done.